### PR TITLE
adapter puppeteer - $ should append previous selector 

### PR DESCRIPTION
**Pull request by mykas**
fixes https://github.com/wix-incubator/unidriver/issues/100

### CODE REVIEW

File: test-suite/src/run-test-suite.ts [hunk 1]
```diff
@@ -202,6 +202,17 @@ export const runTestSuite = (params: TestSuiteParams) => {
                     assert.deepEqual(await driver.$('.todo-item').attr('data-value'), '');
                 });
             });
+
+            it('should get correct attribute [when] $ is deep nested', async () => {
```
Comment (GabiGrin) [inline test-suite/src/run-test-suite.ts:206]:
```suggestion
            it('gets correct attribute [when] $ is deep nested', async () => {
```
(http://bretthard.in/post/tests-should-not-begin-with-should)

File: test-suite/src/run-test-suite.ts [hunk 2]
```diff
@@ -202,6 +202,17 @@ export const runTestSuite = (params: TestSuiteParams) => {
...
+              await runTest({items: [itemCreator({ label: 'Bob' })]}, async (driver) => {
+
+                  const item = await driver.$('.todo-item');
+                  assert.equal(await item.$('.label').text(), 'Bob');
```
Comment (GabiGrin) [inline test-suite/src/run-test-suite.ts:210]: is this needed?

**Comment by GabiGrin** [comment]
thanks @mykas!
looks like other places are also affected by the same bug, can you try adding more tests on places that break?

File: test-suite/src/run-test-suite.ts (hunk 2 above)
Comment (mykas) [reply test-suite/src/run-test-suite.ts:210]: That just making sure, but it can be removed

**Pull request merged**